client/src/components/PortfolioTable.tsx

```diff
@@ -696,34 +697,36 @@ const PortfolioTableWithoutI18n = React.memo((props: PortfolioTableProps) => {
               </span>
             </div>
             <div className="prev">
-              <button
-                className="page-btn"
+              <Button
```

Comment:
currently the disabled state has a drop shadow on hover. is this expected? here's the [figma](https://www.figma.com/file/WZ5AOU0jwtV9PH2d314Sf0/JustFix-Design-System?type=design&node-id=1296%3A1228&mode=dev) i'm looking at for reference